Apps/Playground/iOS/ViewController.swift

```diff
@@ -29,26 +27,24 @@ class UIBabylonGestureRecognizer: UIGestureRecognizer {
         super.touchesBegan(touches, with: event)
         
         for touch in touches {
-            let deviceSlot = _activeTouchIds.firstIndex(of: -1)
-            
-            if (deviceSlot != nil) {
-                _activeTouchIds[deviceSlot!] = touch.hash
-                let loc = touch.location(in: view)
-                
-                _onTouchDown(Int32(deviceSlot!), Int32(loc.x), Int32(loc.y))
-            }
+          if let deviceSlot = _activeTouchIds.firstIndex(of: -1) {
+              _activeTouchIds[deviceSlot] = touch.hash
+              let loc = touch.location(in: view)
+              
+              _onTouchDown(Int32(deviceSlot), Int32(loc.x), Int32(loc.y))
+          }
```

Comment:
nit: Some spacing issues here. Looks like your editor is set to 2, but the file uses 4, so there are some inconsistencies thoughout the file.